src/views/Pools/components/CardFooter.tsx

```diff
@@ -62,6 +67,10 @@ const Row = styled.div`
   display: flex;
 `
 
+const RowRight = styled(Row)`
+  justify-content: flex-end;
+`
+
```

Comment:
Please use the `Flex` component. You can add flex properties as props, e.g. `<Flex justifyContent="flex-end" />`